plugins/linear-genome-view/src/LinearGenomeView/components/Highlight.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useRef, useState } from 'react'
+import { observer } from 'mobx-react'
+import { makeStyles } from 'tss-react/mui'
+import { colord } from '@jbrowse/core/util/colord'
+import { ParsedLocString, getSession } from '@jbrowse/core/util'
+import { Menu } from '@jbrowse/core/ui'
+import { Base1DViewModel } from '@jbrowse/core/util/Base1DViewModel'
+import { IconButton, Tooltip } from '@mui/material'
+
+// icons
+import LinkIcon from '@mui/icons-material/Link'
+
+// locals
+import { LinearGenomeViewModel } from '../model'
+
+type LGV = LinearGenomeViewModel
+
+const COLOR = 'rgb(218, 165, 32)'
```

Comment:
@cmdcolin should this be configurable? could toss it in the settings, or use a theme color